src/OrleansRuntime/OrleansRuntime.csproj

```diff
@@ -119,6 +119,7 @@
     </Compile>
     <Compile Include="ReminderService\MockReminderTable.cs" />
     <Compile Include="Scheduler\ITaskScheduler.cs" />
+    <Compile Include="Startup\ConfigureStartupBuilder.cs" />
```

Comment:
nit.  I'm not sure what this has to do with startup, other than needing to setup the service provider at startup.
I agree we should pull more of this logic out of the silo and into some sort of builder, but why not just move it into the ConfigureServiceBuilder rather than a startup builder.
If there is something I'm not understanding about this being related to 'startup' in general, I'd like to understand that.